apps/web/src/server/app/app.express.js

```diff
@@ -97,18 +95,6 @@ app.use(msalMiddleware);
 // Session middleware
 app.use(session);
 
-// CSRF middleware via session
-app.use(
```

Comment:
removing multer() is enough, but the remaining lines of code of the middleware should be removed too because they break the csrf checking